src/exchange/providers/ollama.py

```diff
@@ -37,7 +37,10 @@ def from_env(cls: Type["OllamaProvider"]) -> "OllamaProvider":
             base_url=url,
             timeout=httpx.Timeout(60 * 10),
         )
-        # from_env is expected to fail if provider is not available
-        # so we run a quick test that the endpoint is running
+        # from_env is expected to fail if required ENV variables are not
+        # available. Since this provider can run with defaults, we substitute
+        # a health check to verify the endpoint is running.
         client.get("")
+        # The OpenAI API is defined after "v1/", so we need to join it here.
+        client.base_url = client.base_url.join("v1/")
```

Comment:
nit: i'd put this above on line 37

Reply:
ill do a quick fix in the 0.9.3 release so we can get this in

Reply:
oh it is subtle. the health check is ollama, so we need to test the base of ollama before appending "v1". It would be better if we didn't need to health check imho, especially as we aren't in any other provider. lemme know your thoughts on that.

Reply:
found a way that may work fine in #60